python/metaparticle_pkg/runner/docker.py

```diff
@@ -0,0 +1,22 @@
+import os
+import subprocess
+
+class DockerRunner:
+    def run(self, img, name, options):
+        # Launch docker container
+        command = ['docker', 'run', '-d', '--name', name]
```

Comment:
What about using `docker-py` for running and building images? Is `subprocess` desired approach or can I work on starting containers using docker API?

Reply:
would love to use docker-py, this is laziness on my part :)

Reply:
@tomkukral that'd be a nice PR all by itself, were you interested in picking up that work?

Reply:
@wagoodman yes, I'd be happy to work on it. PR should be ready at the beginning of the January.